src/handlers/tokenLocksHandler.ts

```diff
@@ -86,6 +86,24 @@ const includeMetagraphUnlockOrdinal = {
   },
 };
 
+const ifActiveDagTokenLock = (event) => {
+  const { active } = event.queryStringParameters || {};
+  return active === "true"
+    ? {
+        dag_token_unlock: null,
+      }
+    : {};
+};
+
+const ifActiveMetagraphTokenLock = (event) => {
```

Comment:
Wherever you can, I would suggest extracting everything from `event` that you need in other functions from within the controller function rather than passing the whole object. That allows the controller to explicitly show what query and path parameters it expects and allows other functions to be much smaller and easier to test since they don't need to know anything about the full request.

Reply:
Agreed, I think we're a good point to do a little reorganization and cleanup of the code.